src/Microsoft.Health.Fhir.SqlServer/Features/Storage/SqlServerFhirModel.cs

```diff
@@ -173,51 +203,42 @@ public void Start()
                     sqlCommand.CommandText = @"
                         SET XACT_ABORT ON
                         BEGIN TRANSACTION
-
                         INSERT INTO dbo.ResourceType (Name) 
                         SELECT value FROM string_split(@resourceTypes, ',')
                         EXCEPT SELECT Name FROM dbo.ResourceType WITH (TABLOCKX); 
-
                         -- result set 1
                         SELECT ResourceTypeId, Name FROM dbo.ResourceType;
-
                         INSERT INTO dbo.SearchParam (Uri)
                         SELECT * FROM  OPENJSON (@searchParams) 
                         WITH (Uri varchar(128) '$.Uri')
-                        EXCEPT SELECT Uri FROM dbo.SearchParam;
-
+                        EXCEPT SELECT Uri FROM dbo.SearchParam
                         -- result set 2
                         SELECT Uri, SearchParamId FROM dbo.SearchParam;
-
                         INSERT INTO dbo.ClaimType (Name) 
                         SELECT value FROM string_split(@claimTypes, ',')
                         EXCEPT SELECT Name FROM dbo.ClaimType; 
-
                         -- result set 3
                         SELECT ClaimTypeId, Name FROM dbo.ClaimType;
-
```

Comment:
I think it's easier to read this with the empty lines between statements:
```sql
SELECT Uri, SearchParamId FROM dbo.SearchParam;
                
INSERT INTO dbo.ClaimType (Name) 
SELECT value FROM string_split(@claimTypes, ',')
EXCEPT SELECT Name FROM dbo.ClaimType; 
```

```sql
SELECT Uri, SearchParamId FROM dbo.SearchParam;
INSERT INTO dbo.ClaimType (Name) 
SELECT value FROM string_split(@claimTypes, ',')
EXCEPT SELECT Name FROM dbo.ClaimType; 
```

Reply:
Oops sorry, I did not mean to change the formatting. This must have been a copy-paste mistake when I was making my new branch. Fixed in c1d9233.